qemu/qemu-guest-agent/pkg.yaml

```diff
@@ -0,0 +1,75 @@
+name: qemu-guest-agent
+variant: scratch
+shell: /bin/bash
+dependencies:
+  - image: debian:{{ .DEBIAN_IMAGE_VERSION }}
```

Comment:
this needs to use the siderolabs toolchain, only exception is that if the app does not work without glbc